.env.example

```diff
@@ -7,3 +7,7 @@ MYSQL_ROOT_PASSWORD=
 
 # Set `WP_ENV` to `yourLastName-local` (ex. `smith-local`).
 WP_ENV=sparkuser-local
+
```

Comment:
I'm a little more inclined to delete the reference to `GOOGLE_API_KEY`, since it isn't actually used for anything. It is useful as an example, though, so maybe we could make it clearer that it is just an example–otherwise I could see someone getting thrown off if they set an API key and nothing changed about the site. Maybe like this?

```php
function define_theme_globals() {
	// phpcs:ignore
	define( 'CUSTOM_ENVIRONMENT_VARIABLE', $_SERVER['CUSTOM_ENVIRONMENT_VARIABLE'] ?? '' );
}
add_filter( 'init', 'define_theme_globals' );
```

I'll create an issue for it, and we can work out the details from there.